### ISSUE TITLE
Make a script to add comments to all Jira issues to indicate that "this was moved to GitHub"

### ISSUE THREAD

**Issue by mocobeta** [apache/lucene-jira-archive#15]
After migration, Jira issues should not be updated.
To prevent updating Jira and guiding people who reach Jira to GitHub, comments should be added to Jira issues that indicate the corresponding GitHub issue URL. The issue mapping will be provided by a CSV file. 

Mapping file format (example):
```
JiraKey,GitHubUrl,GitHubNumber
LUCENE-10605,https://github.com/mocobeta/migration-test-3/issues/37,37
```

Jira comment could be:
```
This was moved to GitHub issue. See <issue url>.
```

**Comment by mikemccand** [comment]
We should also make Jira effectively read-only by editing the workflow for our project to disable any actual changes.  @rmuir had suggested this is possible and indeed sometimes happens by accident ;)

**Comment by mocobeta** [comment]
> We should also make Jira effectively read-only by editing the workflow for our project to disable any actual changes. @rmuir had suggested this is possible and indeed sometimes happens by accident ;)

@mikemccand I know it's technically possible, but I'm unsure if it can be done without additional discussion in the dev@ mail list. This is another issue; please feel free to raise an issue and/or start a discussion in the dev list to gain consensus - and change the Jira workflow - I just have no extra energy to organize a possibly controversial conversation.

Regardless of whether we make Jira read-only or not, I think we should add comments to let readers who reach Jira issues know the issues were moved to GitHub.

**Comment by mikemccand** [comment]
> @mikemccand I know it's technically possible, but I'm unsure if it can be done without additional discussion in the dev@ mail list. This is another issue; please feel free to raise an issue and/or start a discussion in the dev list to gain consensus - and change the Jira workflow - I just have no extra energy to organize another possibly controversial conversation.

OK, I'm sorry for adding stress!  I'll open a separate issue here to discuss / gain consensus.

> Regardless of whether we make Jira read-only or not, I think we should add comments to let readers who reach Jira issues know the issues were moved to GitHub.

+1 -- let's leave this issue open for that purpose.

**Comment by mocobeta** [comment]
Adding a comment: examples
https://developer.atlassian.com/server/jira/platform/jira-rest-api-examples/#adding-a-comment--examples

**Comment by mikemccand** [comment]
It's possible this might still work even once we've made Jira "read-only" by disabling all workflows!  Then we can take our time after the migration (and Jira becoming read-only) to append these comments to all Jira issues.

**Comment by mocobeta** [comment]
Merged #15

Note that this sends notifications to issues@ mail list (per POST API call). I think it's fine to let users know about the migration - an announcement for devs would be needed beforehand though.

This would take about 2.5 hours - I think it'd be okay that we include this in the migration process (the whole process takes days).
